style.css

```diff
@@ -134,4 +209,178 @@
   .icon-3 {
     height: 20px;
   }
+
+  .main_wrapper {
+    display: grid;
+    justify-content: center;
+    justify-items: center;
+    margin: 5%;
+    gap: 1%;
+  }
+
+  .card {
+    display: flex;
+    flex-direction: column;
+    grid-gap: 1.5%;
+    margin-top: 13%;
+    box-sizing: border-box;
+    padding-top: 10%;
+    padding-bottom: 20%;
+    width: 95%;
+    height: auto;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+  }
+
+  h1 {
+    width: 90%;
+    height: 20%;
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #172b4d;
+  }
+
+  h2 {
+    width: 90%;
+    line-height: 92%;
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-weight: 700;
+    font-style: bold;
+    font-size: 40px;
+    display: flex;
+    align-items: center;
+    color: #172b4d;
+    margin-bottom: 0;
+    margin-top: 4%;
+  }
+
+  .text {
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    color: #344563;
+    line-height: 24px;
+    width: 90%;
+    padding: 0 0;
+    margin: 0;
+    font-size: 15px;
+  }
+
+  .project {
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  #color {
+    color: #7f8cff;
+  }
+
+  .social {
+    display: flex;
+    align-self: start;
+    justify-content: space-around;
+    width: 60%;
+    margin: 1% 2%;
+  }
+
+  .text-card {
+    padding-left: 8%;
+  }
+
+  .ul-tag {
+    display: flex;
+    justify-content: space-between;
+    width: 68%;
+    margin: 4% 0;
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    list-style-type: none;
+    padding-left: 0;
+  }
+
+  ul.canopy {
+    display: flex;
+    justify-content: space-between;
+    width: 68%;
+    padding-left: 0;
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-size: 13px;
+    padding-top: 0;
+    list-style-type: disc;
+  }
```

Comment:
- [ ] Excellent job styling the list below your cards 🚀. However, please kindly ensure to style the font bold in order to stick as much as possible to the Figma design🤓.
    > Hint : try applying the CSS property `font-weight: 700;` to the `ul` with class `canopy`